docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,20 @@
 # Staging release notes
 
+## October 2023
+
+**11 October 2023**
+
+Frontend version: 1.0.0-20231009.1523 | Backend version: 1.69.0-20231006.1929
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | Check existing users endpoint on Concierge | With the launch of CAM on 23 October 2023, Concierge will now monitor terminated user events. This triggers the cleanup of users in the TechPass directory and sends webhooks to tenants. However, tenants not subscribed to our webhooks might encounter errors. Without these webhook events, tenants will not be able to tidy up datastores effectively, potentially leading to data inconsistencies. |
+| **Enhancement** | Refactor backend APIs | Significant changes have been made to the *Get Users* API. Additional support for `account:others` and `product:project` has been added, expanding its functionality. Also, the *loadmore* feature has been removed for a more streamlined experience. |
```

Comment:
I thought we are not going to mention **Additional support for `account:others` and `product:project`**?